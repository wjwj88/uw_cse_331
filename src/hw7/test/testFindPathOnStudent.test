# Find a path with Dijkstra's algorithm on student.tsv hw7

# Load student.tsv
LoadGraph student student.tsv

# find path between 
FindPath student Tom Bill
FindPath student Tom Mike