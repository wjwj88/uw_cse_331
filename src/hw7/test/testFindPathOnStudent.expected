# Find a path with Dijkstra's algorithm on student.tsv hw7

# Load student.tsv
loaded graph student

# find path between 
path from Tom to Bill:
Tom to Bill with weight 1.000
total cost: 1.000
path from Tom to Mike:
Tom to Bill with weight 1.000
Bill to Mike with weight 1.000
total cost: 2.000